libecho/src/Packet.cc

```diff
@@ -0,0 +1,117 @@
+#include "Packet.h"
+
+// #define crcpp_uint32
+#include <arpa/inet.h>
+#include "CRC.h"
+
+Packet Packet::loadHeaderFromBytes(const std::vector<uint8_t> &bytes) {
+    Packet p;
+
+    if (bytes.size() != HEADER_SIZE) {
+        throw IncorrectFormat();
+    }
+
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic, cppcoreguidelines-pro-type-cstyle-cast)
+    p.flags = ntohs(((uint16_t *)bytes.data())[0]);
```

Comment:
1) ub
2) nawet jak akceptujemy to ub, to raczej lepiej użyć reinterpret_castu niż tego w stylu C

Reply:
1. Co zrobić żeby nie mieć ub?
2. Ja nie wiem jak tu użyć reinterpret_castu, bo to się skomplikowane robi :v